Resources/views/Admin/Security/Resetting/checkEmail.html.twig

```diff
@@ -27,6 +27,11 @@ file that was distributed with this source code.
     <div class="login-box">
         <div class="login-logo">
             <a href="{{ path('sonata_admin_dashboard') }}">
+                {% if 'single_image' == adminPool.getOption('title_mode') or 'both' == adminPool.getOption('title_mode') %}
+                    <div>
+                        <img style="width:64px;" src="{{ asset(adminPool.titlelogo) }}" alt="{{ adminPool.title }}">
```

Comment:
Uuuuh no way around the `style` attribute? Like a class?

Reply:
Maybe it's time create a `title_logo_size` option?
I'm not proud but I've overidden the whole template because I needed to have a wider logo :)

Reply:
Do we really have to use CSS resize, BTW?

Reply:
If there is a way to delete this `width` and that users who have a logo wider than 64px don't see it at their original size, then please remove it :)

Reply:
`overflow: hidden`?